zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1358,6 +1358,19 @@ Both subsystems need to have sufficient amount of threads to achieve peak read t
     Number of Commit Processor worker threads. If configured with 0 worker threads, the main thread
     will process the request directly. The default value is the number of cpu cores.
 
+* *zookeeper.commitProcessor.maxReadBatchSize* :
```

Comment:
Can you please add some hints on concrete numbers? 
I know much depends on use-case, still, I think some general guide would be nice here. 
Something like "setting to an order of magnitude of 2-9... of 10s is... and setting it to 100+ is only advisable when..."